### PR TITLE
Add meta kernel coverage for aten.unsafe_split, aten.unsafe_chunk

### PR DESCRIPTION
**Pull request by tugsbayasgalan**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #92608

### CODE REVIEW

File: torch/_meta_registrations.py [hunk 1]
```diff
@@ -260,6 +260,42 @@ def linalg_cholesky_ex(A: Tensor, upper: bool = False, check_errors: bool = Fals
     return L, infos
 
 
+@register_meta(aten.unsafe_split_with_sizes.default)
```
Comment (ngimel) [inline torch/_meta_registrations.py:263]: can we reuse `split_with_sizes` decomposition and not add separate meta?

File: torch/_refs/__init__.py [hunk 2]
```diff
@@ -2560,6 +2560,18 @@ def broadcast_tensors(*tensors) -> List[TensorLikeType]:
     return list(_maybe_broadcast(*tensors, preserve_cpu_scalar_tensors=False))
 
 
+@aten.unsafe_chunk.default.py_impl(DispatchKey.CompositeImplicitAutograd)
```
Comment (ngimel) [inline torch/_refs/__init__.py:2563]: why does this go to _refs?

Comment (tugsbayasgalan) [reply torch/_refs/__init__.py:2563]: This was suggested by @ezyang, I am not too sure about the reasoning :)

File: torch/_meta_registrations.py (hunk 1 above)
Comment (tugsbayasgalan) [reply torch/_meta_registrations.py:263]: Hmm do you know where I can find it? Seems like _meta_registrations.py doesn't have any reference to split_with_sizes.

File: torch/_refs/__init__.py (hunk 2 above)
Comment (ezyang) [reply torch/_refs/__init__.py:2563]: oh oops, I either told you the wrong thing, or it got garbled in transit. Put it in _decompositions.

Comment (tugsbayasgalan) [reply torch/_refs/__init__.py]: ok updated :)

File: torch/_meta_registrations.py (hunk 1 above)
Comment (ngimel) [reply torch/_meta_registrations.py:263]: As far as I can see, you can also register this decomp https://github.com/pytorch/pytorch/blob/0998ec1e27b9d929275d43d324dd9342409f705c/torch/_decomp/decompositions.py#L1089 to `unsafe_split_with_sizes` and it should work

Comment (tugsbayasgalan) [reply torch/_meta_registrations.py]: ohh i see. Just updated to reflect this

**Review by ngimel — APPROVED** [review]

**Comment by ngimel** [comment]
@tugsbayasgalan you need to remove xfail from unexpectedly passing test before relanding

**Pull request closed without merging**